cmd/yaml.go

```diff
@@ -33,6 +38,46 @@ func ParseYaml(c cookoo.Context, p *cookoo.Params) (interface{}, cookoo.Interrup
 	return cfg, nil
 }
 
+// LookupManifestFile look up for a given file
+// in the current working directory
+func LookupManifestFile(fname string, p *cookoo.Params) (interface{}, cookoo.Interrupt) {
```

Comment:
I've just pushed a new commit which should look first at `.yaml`, and then to `.yml`, so now you're right. Having multiple files with the same name a different extension is an edge scenario and a mistake by the end user.